tests/foreman/destructive/test_capsulecontent.py

```diff
@@ -0,0 +1,111 @@
+"""Capsule-Content related tests, which require destructive Satellite
+
+:Requirement: Capsule-Content
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: Capsule-Content
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_alpha
+
+from robottelo import constants
+
+pytestmark = [pytest.mark.destructive]
+
+
+@pytest.mark.tier4
+@pytest.mark.skip_if_not_set('capsule')
+def test_positive_sync_without_deadlock(
```

Comment:
I dont see case importance changed for this test so its `high`, are you agree with that ?

Reply:
The importance of the whole module is set to `high`. Isn't it enough? Do we need to state the importance for every single test explicitly?

Reply:
@ogajduse No, I mean it is not about inheriting the importance from module level but I believe this could come under `Critical` severity as it belongs to capsule syncing and thats one of the vital thing capsule is made for.

So my real question is, would you like to change the importance to critical for this test ?

Reply:
I just respected the severity defined for the BZ. I don't oppose to set it higher, but maybe I would rather do that for some basic scenario like RH or KS repos sync to capsule in `tests/foreman/api/test_capsulecontent.py`, than this deadlock one.